api/data/surveys/transform.ts

```diff
@@ -0,0 +1,100 @@
+import * as dataForge from 'data-forge';
+import * as fs from 'node:fs';
+import { CountryISO3Map } from '@shared/constants/country-iso3.map';
+import { StringUtils } from '@api/utils/string.utils';
+
+const main = async () => {
```

Comment:
Please specify that this utility is only to load mock initial data 

(I hope that's the only purpose of this xD)

Reply:
Additionally, when this is a bit more mature in the near future, we can have just a preprocessed json/object whatever that can be loaded